map.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+	'use strict';
+	let mymap;
+	kintone.events.on('app.record.detail.show', function(event) {
+		const record = event.record;
+		const address = record.Address.value;
+		const headerSpace = kintone.app.record.getHeaderMenuSpaceElement();
+		const cybozuAddress = [35.68704, 139.7861];
+		const name = record.Place.value;
+
+		headerSpace.style.height = '300px';
+		headerSpace.setAttribute('id', 'nihonbashi');
```

Comment:
Please avoid applying styles or adding the id attribute and to kintone's baked-in element as much as possible because it's DOM manipulation. Customization with DOM manipulation might be affected by kintone updates. For example, if the id attribute is added in the header space element later on kintone updates, your customization may affect how the header space behaves.
Create an element (e.g. `<div>` or `<span>`) with `document.createElement()` or `$('<tagName>')`, applying styles to it, adding the id attribute, and append it to the header space.